integration/contractdeployer/contract_deployer_test.go

```diff
@@ -58,3 +134,48 @@ func createObscuroNetwork(t *testing.T) {
 		panic(fmt.Sprintf("failed to create test Obscuro network. Cause: %s", err))
 	}
 }
+
+// Returns a viewing-key client with a registered viewing key.
+func getClient(wallet wallet.Wallet) *rpcclientlib.EncRPCClient {
+	viewingKey, err := rpcclientlib.GenerateAndSignViewingKey(wallet)
+	if err != nil {
+		panic(err)
+	}
+	client, err := rpcclientlib.NewEncNetworkClient(nodeAddress, viewingKey)
+	if err != nil {
+		panic(err)
+	}
+	return client
+}
+
+// Prefunds the contract deployer with the given amount.
+func prefundContractDeployer(faucetWallet wallet.Wallet, faucetClient *rpcclientlib.EncRPCClient, prealloc int64) {
```

Comment:
there's nothing really ContractDeployer specific about this method that I can see - worth making this generic for funding an address/private key?

Reply:
Sure. Extracted the method used in the sims and used that here as well.